topics/outcomes-metrics.yaml

```diff
@@ -0,0 +1,16 @@
+topic: outcomes-metrics
+pipeline: outcomes
```

Comment:
Is this correct?

Reply:
I think this is used primarily to identify a pipeline logically. @lynnagara  and @untitaker  can provide more context.
But it should be correct as we call `outcomes` everything that populates the outcomes and billing data,

Reply:
I assume the metrics outcomes eventually makes it's way to the shared outcomes table in clickhouse with all the other outcomes? So this pipeline is probably fine

Reply:
It will be parallel to the current outcomes with their own schema and from the looks of it in a separate clickhouse cluster.

Reply:
If everything is fully separate, I think you can change this to `outcomes-metrics` or something like that